attestation-agent/kbs_protocol/src/bin/trustee-agent/README.md

```diff
@@ -0,0 +1,23 @@
+Trustee agent -- a simple tool to attest and fetch secrets from Trustee
+
+Trustee agent is using attestation-agent's kbs_protocol client and
+attesters to gather hardware-based confidential-computing evidence
+and send it over to Trustee.
+
+Trustee agent is a part of [confidential-containers](https://github.com/confidential-containers)
+[guest-components](https://github.com/confidential-containers/guest-components)
+project but can be used for confidential VMs as well.
+
+
+
+Build with:
+    cargo build -p kbs_protocol --bin trustee-agent --no-default-features \\
+          --features "background_check,passport,<openssl|rusttls>,bin,<attesters-list>"
```

Comment:
```suggestion
This can be formatted into a markdown style

Build with:
```bash
cargo build -p kbs_protocol --bin trustee-agent --no-default-features \
          --features "background_check,passport,<openssl|rusttls>,bin,<attesters-list>"
```
```

Reply:
Thanks for the suggestion. I added this and some headers